### PR TITLE
RadioButton style

### PR DESCRIPTION
**Pull request by EAlexisDuque**
**Solution**
ezgif com-video-to-gif

### CODE REVIEW

File: app/src/main/java/co/condorlabs/customcomponents/customradiogroup/BaseRadioGroupFormField.kt [hunk 1]
```diff
@@ -30,21 +29,20 @@ import co.condorlabs.customcomponents.formfield.Selectable
 import co.condorlabs.customcomponents.formfield.ValidationResult
 import co.condorlabs.customcomponents.helper.*
 
-abstract class BaseRadioGroupFormField(context: Context, private val mAttrs: AttributeSet) :
-    TextInputLayout(context, mAttrs), FormField<String> {
+abstract class BaseRadioGroupFormField(
+    context: Context, private val mAttrs: AttributeSet
+) : TextInputLayout(context, mAttrs), FormField<String> {
```
Comment (ferazog) [inline app/src/main/java/co/condorlabs/customcomponents/customradiogroup/BaseRadioGroupFormField.kt]: `mAttrs` -> `attrs`

File: app/src/main/java/co/condorlabs/customcomponents/customradiogroup/BaseRadioGroupFormField.kt [hunk 2]
```diff
@@ -30,21 +29,20 @@ import co.condorlabs.customcomponents.formfield.Selectable
...
-    private var mLabelText = EMPTY
-
-    private val mLayoutParams = LinearLayout.LayoutParams(
-        LinearLayout.LayoutParams.MATCH_PARENT,
-        LinearLayout.LayoutParams.WRAP_CONTENT
+    private var selectables: List<Selectable>? = null
+    private var radioGroup: RadioGroup? = null
+    private var labelText = EMPTY
+    private var spaceBetweenItems = DEFATUL_SPACE_BETWEEN_ITEMS
+    private val mLayoutParams = LayoutParams(
```
Comment (ferazog) [inline app/src/main/java/co/condorlabs/customcomponents/customradiogroup/BaseRadioGroupFormField.kt]: `mLayoutParams` -> `layoutParams`

File: app/src/main/java/co/condorlabs/customcomponents/customradiogroup/BaseRadioGroupFormField.kt [hunk 3]
```diff
@@ -30,21 +29,20 @@ import co.condorlabs.customcomponents.formfield.Selectable
...
-    private var mRadioGroup: RadioGroup? = null
-    private var mLabelText = EMPTY
-
-    private val mLayoutParams = LinearLayout.LayoutParams(
-        LinearLayout.LayoutParams.MATCH_PARENT,
-        LinearLayout.LayoutParams.WRAP_CONTENT
+    private var selectables: List<Selectable>? = null
+    private var radioGroup: RadioGroup? = null
+    private var labelText = EMPTY
+    private var spaceBetweenItems = DEFATUL_SPACE_BETWEEN_ITEMS
```
Comment (ferazog) [inline app/src/main/java/co/condorlabs/customcomponents/customradiogroup/BaseRadioGroupFormField.kt]: `DEFATUL...` -> `DEFAULT...`

File: app/src/main/java/co/condorlabs/customcomponents/customradiogroup/BaseRadioGroupFormField.kt [hunk 4]
```diff
@@ -94,18 +93,17 @@ abstract class BaseRadioGroupFormField(context: Context, private val mAttrs: Att
...
+        tvLabel.text = labelText
+        addView(tvLabel, mLayoutParams)
+        radioGroup = RadioGroup(context, mAttrs).apply {
             id = R.id.rgBase
         }
 
-        mRadioGroup?.setOnCheckedChangeListener { _, checkedId ->
-
-            mSelectables?.forEach { it.value = false }
+        radioGroup?.setOnCheckedChangeListener { _, checkedId ->
```
Comment (ferazog) [inline app/src/main/java/co/condorlabs/customcomponents/customradiogroup/BaseRadioGroupFormField.kt]: `checkedId` could be replaced by `_`

File: app/src/main/java/co/condorlabs/customcomponents/helper/Constants.kt [hunk 5]
```diff
@@ -112,3 +112,6 @@ typealias ButtonType = String
 /** FONT PATHS **/
 const val OPEN_SANS_SEMI_BOLD = "fonts/OpenSans-SemiBold.ttf"
 const val OPEN_SANS_REGULAR = "fonts/OpenSans-Regular.ttf"
+
+/** RADIO GROUP **/
+const val DEFATUL_SPACE_BETWEEN_ITEMS = 10
```
Comment (ferazog) [inline app/src/main/java/co/condorlabs/customcomponents/helper/Constants.kt]: add new line below

File: app/src/main/res/values/styles.xml [hunk 6]
```diff
@@ -24,6 +24,15 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="radio_button_custom_style" parent="Widget.AppCompat.CompoundButton.RadioButton">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:foreground">?attr/selectableItemBackgroundBorderless</item>
+        <item name="android:paddingStart">16dp</item>
+        <item name="android:paddingEnd">16dp</item>
```
Comment (ferazog) [inline app/src/main/res/values/styles.xml]: `android:paddingStart` and `android:paddingEnd` could be called  from a dimen resource.

File: app/src/main/java/co/condorlabs/customcomponents/customradiogroup/BaseRadioGroupFormField.kt (hunk 4 above)
Comment (EAlexisDuque) [reply app/src/main/java/co/condorlabs/customcomponents/customradiogroup/BaseRadioGroupFormField.kt]: No

**Pull request merged**